toolchain/defs.bzl

```diff
@@ -24,14 +24,14 @@ URL_FORMAT_NIGHTLY = "https://ziglang.org/builds/zig-{host_platform}-{version}.{
 # generous enough to host the artifacts, which we use.
 URL_FORMAT_BAZELMIRROR = "https://mirror.bazel.build/" + URL_FORMAT_NIGHTLY.lstrip("https://")
 
-_VERSION = "0.11.0-dev.1796+c9e02d3e6"
+_VERSION = "0.11.0-dev.2545+311d50f9d"
```

Comment:
This needs to be also updated in mirror.bazel.build. Here is a mirror request from the past: https://github.com/bazelbuild/bazel/issues/17635

```
motiejus@mtwork:~$ curl -I https://ziglang.org/builds/zig-linux-x86_64-0.11.0-dev.2545+311d50f9d.tar.xz
HTTP/2 200 
content-type: application/x-xz
content-length: 44252188
date: Tue, 11 Apr 2023 10:02:54 GMT
x-amz-meta-s3cmd-attrs: atime:1681201277/ctime:1681200608/gid:1000/gname:ci/md5:89a67c9f8df75eeb80955cb553892a28/mode:33188/mtime:1681200608/uid:1000/uname:ci
cache-control: public, max-age=31536000, immutable
last-modified: Tue, 11 Apr 2023 08:21:49 GMT
etag: "857fe2f62b85b683d745714a4e6ce423-3"
server: AmazonS3
x-cache: Hit from cloudfront
via: 1.1 2fe761c42f710dbc97bfbe41f450bf42.cloudfront.net (CloudFront)
x-amz-cf-pop: ARN54-C1
x-amz-cf-id: o-IURQdV21K7WCei88sd0ahhyYIRulBB-q-6sKcF9IO7dBm2GvqOow==
age: 3948

motiejus@mtwork:~$ curl -I https://mirror.bazel.build/ziglang.org/builds/zig-linux-x86_64-0.11.0-dev.2545+311d50f9d.tar.xz
HTTP/2 404 
x-guploader-uploadid: ADPycds1gh-8QsfM3gq0d11VQFRAWdVltRSoc9lzSTtMMR6XgCp3UBbOwQlBheQaSkDFzr9wqA-0l8atFi0C-CG_gEB-sQ
content-type: application/xml; charset=UTF-8
content-length: 127
date: Tue, 11 Apr 2023 11:08:46 GMT
expires: Tue, 11 Apr 2023 11:08:46 GMT
cache-control: private, max-age=0
server: UploadServer
alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000

motiejus@mtwork:~$ 
```